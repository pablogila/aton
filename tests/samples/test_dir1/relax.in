
&CONTROL
  calculation   = 'relax'
  pseudo_dir    = './pseudos/'
  restart_mode  = 'from_scratch'
  max_seconds   = 1000
  etot_conv_thr = 1.0d-12
  forc_conv_thr = 1.0d-07
  nstep         = 100
/

&SYSTEM
  ecutwfc     = 60.0
  ibrav       = 1
  A           = 10.0
  nat         = 3
  ntyp        = 3
  input_dft   = 'PBEsol'
  occupations = 'fixed'
/

&ELECTRONS
  conv_thr         = 2.0d-12
  diagonalization  = 'ppcg'
  electron_maxstep = 100
/

&IONS
  ion_dynamics = 'bfgs'
  upscale      = 100
/

K_POINTS (automatic)
  2 2 2 0 0 0

CELL_PARAMETERS {alat}
  2.000000000000000   0.000000000000000   0.000000000000000 
  0.000000000000000   2.000000000000000   0.000000000000000 
  0.000000000000000   0.000000000000000   2.000000000000000 
ATOMIC_SPECIES
   I  126.90400   I.upf
   N   14.00650   N.upf
   C   12.01060   C.upf
ATOMIC_POSITIONS {crystal}
 I   5.000000000000000   0.000000000000000   0.000000000000000 
 C   0.000000000000000   5.000000000000000   0.000000000000000 
 N   0.000000000000000   0.000000000000000   5.000000000000000 
