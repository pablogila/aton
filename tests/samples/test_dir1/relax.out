This is a dummy file to test value extraction from Quantum ESPRESSO output files.
------------------------------------------------------------------------
Bad values that should NOT be reported:
     Total force =     0.000002     Total SCF correction =     0.000002
!    total energy              =   -2000.00000000 Ry   ! Bad value
------------------------------------------------------------------------
Good values that should be reported:
     bfgs converged in  97 scf cycles and  95 bfgs steps

Begin final coordinates
     new unit-cell volume =   2.0 a.u.^3 (   925.14878 Ang^3 )
     density =      1.0 g/cm^3

CELL_PARAMETERS (alat= 10.0000)
   1.000000000   0.000000000   0.000000000 
   0.000000000   1.000000000   0.000000000 
   0.000000000   0.000000000   1.000000000 

ATOMIC_POSITIONS (crystal)
I                1.0000000000        0.0000000000        0.0000000000 
C                0.0000000000        1.0000000000        0.0000000000 
N                0.0000000000        0.0000000000        1.0000000000 
End final coordinates

!    total energy              =   -1000.00000000 Ry

     random text in between
     atom   47 type  2   force =    -0.00002093    0.00000625   -0.00000000
     atom   48 type  2   force =    -0.00002093   -0.00000625    0.00000000

     Total force =     0.000001     Total SCF correction =     0.000000

     Parallel routines

     PWSCF        :  48m 8.71s CPU  48m47.56s WALL


   This run was terminated on:  23:42:31   8Sep2024            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=

